# 异常

## 异常处理

```
try:
    val = dic[7]
except (KeyError,TypeError) as erro:  #或者except Exception as e:
    print(" : %s"%erro)
else:
    无任何异常时执行
```

## 抛出异常
raise Exception("抛出一个异常")  